meta-resin-common/recipes-connectivity/networkmanager/networkmanager_1.8.0.bb

```diff
@@ -51,7 +51,7 @@ PACKAGECONFIG[systemd] = " \
 PACKAGECONFIG[bluez5] = "--enable-bluez5-dun,--disable-bluez5-dun,bluez5"
 # consolekit is not picked by shlibs, so add it to RDEPENDS too
 PACKAGECONFIG[consolekit] = "--with-session-tracking=consolekit,,consolekit,consolekit"
-PACKAGECONFIG[concheck] = "--with-libsoup=yes,--with-libsoup=no,libsoup-2.4"
+#PACKAGECONFIG[concheck] = "--with-libsoup=yes,--with-libsoup=no,libsoup-2.4"
```

Comment:
@floion The idea of bb and bbappend for this recipe was to keep the bb recipe as close as possible to the pristine one from meta-resin. Commenting out the list makes this change clearer in the future when somebody will need to diff the original and our recipe.